pyjuicenet/__init__.py

```diff
@@ -196,6 +239,35 @@ async def get_info(self, charger: Charger):
         )
         return await response.json()
 
+    async def get_commands(self, charger: Charger):
+        """Fetch more info about the charger."""
+        data = {
+            "cmd": "list_commands",
+            "device_id": self.uuid,
+            "token": charger.token,
+            "account_token": self.api_token
+        }
+
+        response = await self.session.post(
+            f"{BASE_URL}/box_api_secure", json=data,
+        )
+        return await response.json()
```

Comment:
Not sure where this API command comes from?
It doesn't work for me at all?

Reply:
Oops, sorry about that. That was a remnant from me trying to fuzz the API to see if there was anything we didn't know about. The short answer is that there isn't (I let it run for days, had to throttle it, and no new endpoints came up). I pushed an update with that function removed.